src/indexApi.ts

```diff
@@ -98,7 +98,7 @@ async function main(): Promise<void> {
     }
     const { start, end, appId, pluginIds } = analyticsQuery
     const timePeriod = analyticsQuery.timePeriod.toLowerCase()
-    const queryOffset = timePeriod === 'day' ? SIX_DAYS_IN_MINUTES : 0
+    const queryOffset = timePeriod.includes('day') ? SIX_DAYS_IN_MINUTES : 0
```

Comment:
Constant should probably be renamed to `SIX_DAYS_IN_SECONDS`. The calculation on line 11 is 6 days x 24 hours/day x 60 minutes / hour x 60 seconds / minutes = 518400 seconds.

Reply:
Fixed